apps/web/src/routes/_app/user.tsx

```
@@ -1,26 +1,186 @@
+                  fr: 'Numéro de téléphone'
+                }),
+                variant: 'input'
+              },
+              email: {
+                kind: 'string',
+                label: t({
+                  en: 'Email',
+                  fr: 'Courriel'
+                }),
+                variant: 'input'
+              }
```

ESLint

[error] Expected "email" to come before "phoneNumber".

(perfectionist/sort-objects)